res/presets/Gainful Clipper.chowpreset

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<Preset name="Gainful Clipper" plugin="BYOD" vendor="CHOW" category="Pedals"

```

Comment:
If you'd prefer to put your own name as the vendor, that would be cool as well. Totally up to you!